include/mega/utils.h

```diff
@@ -222,6 +222,90 @@ struct MEGA_API MemAccess
 int mega_snprintf(char *s, size_t n, const char *format, ...);
 #endif
 
+struct MEGA_API TLVstore
+{
+private:
+    TLV_map tlv;
+
+ public:
+
+    /**
+     * @brief containerToTLVrecords Builds a TLV object with records from an encrypted container
+     * @param data Binary byte array representing the encrypted container
+     * @param datalen Length of the byte array.
+     * @param key Master key to decrypt the container
+     * @return A new TLVstore object. You take the ownership of the object.
+     */
+    static TLVstore * containerToTLVrecords(const string *data, SymmCipher *key);
+
+    /**
+     * @brief Builds a TLV object with records from a container
+     * @param data Binary byte array representing the TLV records
+     * @param datalen Length of the byte array.
+     * @return A new TLVstore object. You take the ownership of the object.
+     */
+    static TLVstore * containerToTLVrecords(const string *data);
+
+    /**
+     * @brief Converts the TLV records into an encrypted byte array
+     * @param key Master key to decrypt the container
+     * @param mode Block encryption mode to be used by AES
+     * @return A new string holding the encrypted byte array. You take the ownership of the string.
+     */
+    string *TLVrecordsToContainer(SymmCipher *key, encryptionmode_t mode = AES_GCM_12_16);
```

Comment:
I don't like a function starting with an upper case letter. I just thought this was the constructor of a class ...

Methods and functions should *always* start with a lower case letter!